### ISSUE TITLE
Sybil Report 2 (whale that no one has discovered)

### ISSUE THREAD

**Issue by DespiteDeath1** [connext/community-sybil-reports#459]
# Related Addresses
0xec21c3919da7062a8eb66d22629347db8ce94a1b
0x9e203460cd7cfa6b1b3df4dcbfbb896f29fb73a7
0xf54444bea21225141057022d7593c42d298aedbd
0x18bdd6f71341c8bc169aa734647df0638a684a0a
0xeb372fc0a47cd3be455f102fff4484d5d2e4c16d
0x280c5201be36fbd93e091393aa54c76ab1057978
0xf44c5f75c4eeb6433a285b6b05765343a1dd72e6
0xb397bd1d5879ad7628975d6dc40ffbd970bb76e9
0xf5df6526b3bb27520f43861d1e7a0bb431fa10b2
0x08e5b30235b2e30143e1b93cb170b69358a52c5e
0x55d4ba59a84fe93d91b1f6fc48aa02255ac75492
0x327ba2b8cad057996179bb348fee853282394f5e
0xd3636aec7782ac9ed00dea96ac82636c23b050ca
0x14c76fb075a65e23b940f286f41a5137bb520848
0x26f0519442a4a349c22e150f942da4c3d6331803

# Reasoning
I used [Dune](https://dune.com/KARTOD/connext-bridge-activity-and-top-30percent-depositors) and [DeBank](https://debank.com/) to detect all this Sybils.
I started going through the wallets with Dune. After that I found 2 very similar wallets. I checked the latest activity in Arbitrum and sure enough it was Sybil. But I realized that it probably wasn't all of his wallets, so I started thinking. Our "genius" (Sybil) made our job easier because on DeBank you can track who he is subscribed to. And I found one of the main wallets of this user 0x3f7c411dd1d404c1cf92409ba6dadcbce9dca4f0 . And this wallet itself is already subscribed to 19 other such wallets (His own).
![image](https://github.com/connext/community-sybil-reports/assets/143283408/a08e66f6-8664-493d-8995-020b9cc60d67)
![image](https://github.com/connext/community-sybil-reports/assets/143283408/b9b5cc7c-f980-41f5-83df-366cc28cc73d)
After that, the only thing left is to make sure that all these accounts belong to the same person. It is very difficult to find intersections between wallets, because mostly sybils use sub-accounts of exchanges, so that they would not be uncovered.
But he still got caught, in several places, which will prove it was all 1 person.

# Methodology
1) Transactions on the wallets happen at the same time and on the same platforms (Stargate, Uniswap, Trade Joe). Yes, these are popular platforms, but swaps and breeches are made at the same time on all wallets in a difference of 2-3 minutes.
An example of two of them:
![image](https://github.com/connext/community-sybil-reports/assets/143283408/1258cbb7-cbc6-43e2-961c-57879af94500)

![image](https://github.com/connext/community-sybil-reports/assets/143283408/360bd1ed-84f6-4e4d-af06-044796adb647)

2) All wallets from my list, Subscribed to 2 wallets in Debank (you can think of them as overlapping with each other)
https://debank.com/profile/0x3f7c411dd1d404c1cf92409ba6dadcbce9dca4f0.
https://debank.com/profile/0x16d69a835c012584783921b2e4c622f157ce97bf
![image](https://github.com/connext/community-sybil-reports/assets/143283408/dccb9571-9ee9-407b-887c-2daf39c805d5)
3)All wallets have funds on the same networks. ARB + ZkSYNC
![image](https://github.com/connext/community-sybil-reports/assets/143283408/f3a9e8ec-9457-43fb-b59c-694ad76f305f)

![image](https://github.com/connext/community-sybil-reports/assets/143283408/0068bead-bdbd-4d83-8ea8-5fab5ec8b3ae)

![image](https://github.com/connext/community-sybil-reports/assets/143283408/365d2d8e-b997-4223-8792-9d6d4cc3947c)

Bottom line: All wallets perform the same operations at the same time and have overlap with each other in the form of subscriptions to 2 other wallets. 

# Rewards Address

0x44ca02D6E80A44dc3b2be8b09f88309F74c42F9B

**Comment by ronaskin** [comment]
![Screenshot_10](https://github.com/connext/community-sybil-reports/assets/137379395/26c21b1b-7f3c-45d0-910d-b64fb2c905ea)

**Comment by DespiteDeath1** [comment]
The second Sybil, I only listed four wallets. He has many more, but they have already been discovered by other users 
https://github.com/connext/community-sybil-reports/issues/72
